supplier/html-splitter/src/test/java/io/knotx/fragments/supplier/html/splitter/HtmlAttributesParserTest.java

```diff
@@ -18,15 +18,32 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import io.knotx.junit5.util.FileReader;
+import java.io.IOException;
 import java.util.List;
 import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class HtmlAttributesParserTest {
 
+  private static final String SOME_KEY = "some-parameter";
+
+  private static final String SINGLE_QUOTES_ESCAPED = "\\' Some text \\'having\\' single quotes escaped \\'\\' and some \"double quotes\" \\'";
+  private static final String DOUBLE_QUOTES_ESCAPED = "\\\" Some text \\\"having\\\" double quotes escaped '' and some 'single quotes' \\\"";
+
   private HtmlAttributesParser tested;
 
+  private static String json;
```

Comment:
`json` and `bas64` don't have to be static.

Reply:
Done